### DISCUSSION TITLE
Routr & Homer intergration

### DISCUSSION THREAD

**Discussion by elianero** [fonoster/routr#227] [General]
Hello!

I was wondering if Routr has integration with HEP (Homer) or any other platform to capture traffic QOS/CDR?

Thank you in advance!

Regards,
George

**Comment by psanders** [comment]
We actually added experimental support for HEP yesterday. We don't have much documentation yet, but feel free to dig into the release notes: https://github.com/fonoster/routr/releases/tag/v2.3.0

**Comment by elianero** [comment]
Great news! Thank you for the reply!

**Reply by psanders** [reply]
Any time. Please consider joining the community at Discord and give us a ⭐
